src/databricks/labs/ucx/installer/workflows.py

```diff
@@ -364,6 +397,9 @@ def create_jobs(self, prompts):
     def _config_file(self):
         return f"{self._installation.install_folder()}/config.yml"
 
+    def _is_testing(self):
+        return self._product_info.product_name() != "ucx"
```

Comment:
Does mypy complain here? In the test the `product_name` look to be `None`

Reply:
this may not work since blueprint infer the product_name based on the version folder if `product_name` is `None`
```
        if self._product_name:
            return self._product_name
        version_file_folder = self._version_file.parent
        return version_file_folder.name.replace("_", "-")
```